modules/developer-lightspeed/con-supported-architecture.adoc

```diff
@@ -3,13 +3,14 @@
 [id="con-supported-architecture_{context}"]
 = Supported architecture for {ls-brand-name}
 
-{ls-short} is available as a plugin on all platforms that host {product-very-short}, and it requires the use of {rcs-name} ({rcs-short}) as a sidecar container.
+{ls-short} is available as a plugin on all platforms that host {product-very-short}. It requires the use of {lcs-name}, which runs as a sidecar container. {lcs-short} acts as the intermediary layer for interfacing with Large Language Model (LLM) providers.
 
+{ls-short} handles functionalities like question validation, feedback, and Retrieval-Augmented Generation (RAG) by leveraging components often built with Llama Stack.
```

Comment:
I would say leave this sentence out as we already discussed this in the section modules/developer-lightspeed/con-about-lightspeed-stack-and-llama-stack.adoc